### PR TITLE
Start screen

### PR DESCRIPTION
**Pull request by helomdrs**
The intial screen of the game that consists in:

- A grid with two tilemaps as a background art turned into a prefab;
- A Canvas object with:

1. A TextMesh PRO for the title of the project;
2. An Image of the main character with a simple idle animation;
3. A Start Button that when pressed, disable itself and enable a panel;
4. A panel with the three buttons of difficulty to be chosen and the [Difficulty Selector script](https://github.com/helomdrs/Flappy-Bird/compare/development...start-screen#diff-b05394301388e49329cd4288e5c3223a03c9691ca29d6c13e9a5ef9b2c015a1d);

- An empty object that holds the [Scene Loader script](https://github.com/helomdrs/Flappy-Bird/compare/development...start-screen#diff-a7b7e16b6ddd2d2b488e2b26f0dded6d127cdf415ca8373d2453c2938d0ceb89);
- An empty object that holds the [Difficulty Holder script](https://github.com/helomdrs/Flappy-Bird/compare/development...start-screen#diff-1775b81d8eafdd0d1f96b4ffdf0732fe607627895788ed1fe55ef7a122b49835);

Other changes:

- Two spritesheets into tilemaps;
- An Animator and one animation for the character;
- Two custom fonts assets;

### CODE REVIEW

File: Assets/Scripts/DifficultySelector.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DifficultySelector : MonoBehaviour
+{
+    public void SetDifficultySelected(int diffilcutySelected)
+    { 
+        if (diffilcutySelected > 3 | diffilcutySelected < 1) diffilcutySelected = 2;
```
Comment (psydack) [inline Assets/Scripts/DifficultySelector.cs]: Why are you using bitwise instead short circuit?

File: Assets/Scripts/ScenesLoader.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class ScenesLoader : MonoBehaviour
+{
+    public void LoadStartScene()
+    {
+        SceneManager.LoadScene("StartScene", LoadSceneMode.Single);
```
Comment (psydack) [inline Assets/Scripts/ScenesLoader.cs]: suggestion: Convert the scenes names to a constant to easily change.

File: Assets/Scripts/DifficultySelector.cs (hunk 1 above)
Comment (mariotoledo) [inline Assets/Scripts/DifficultySelector.cs]: Aqui era pra sert um or, né? No caso, são, o or no C# são duas barrinhas ||

**Review by mariotoledo — CHANGES_REQUESTED** [review]

File: Assets/Scripts/DifficultySelector.cs (hunk 1 above)
Comment (helomdrs) [reply Assets/Scripts/DifficultySelector.cs]: Era ser um or mesmo opss

Comment (psydack) [reply Assets/Scripts/DifficultySelector.cs]: Already answered

File: Assets/Scripts/ScenesLoader.cs (hunk 2 above)
Comment (helomdrs) [reply Assets/Scripts/ScenesLoader.cs]: done in the [new commit](https://github.com/helomdrs/Flappy-Bird/pull/1/commits/fee3fb2c5590695be0535d9d3a7a0ffda38c6286)

**Review by psydack — APPROVED** [review]
suggestion: You can add .vscode in gitignore.

**Pull request merged**